kernel.spec

```diff
@@ -53,6 +53,14 @@ BuildRequires:  bc
 BuildRequires:  openssl-devel
 BuildRequires:  gcc-plugin-devel
 BuildRequires:  elfutils-libelf-devel
+
+# gcc with support for BTI mitigation
+%if 0%fedora == 23
```

Comment:
Better use `%{?fedora}`, otherwise spec _parsing_ will fail if there is no `%fedora` set.

Reply:
Fixed.